pkg/controller/computenodeopenstack/computenodeopenstack_controller.go

```diff
@@ -133,20 +140,84 @@ func (r *ReconcileComputeNodeOpenStack) Reconcile(request reconcile.Request) (re
 		return reconcile.Result{}, err
 	}
 
+	// ScriptsConfigMap
+	scriptsConfigMap := computenodeopenstack.ScriptsConfigMap(instance, strings.ToLower(instance.Kind)+"-scripts")
+	if err := controllerutil.SetControllerReference(instance, scriptsConfigMap, r.scheme); err != nil {
```

Comment:
shouldn't this be done inside the  IsNotFound if? only adding the reference when creating it...

Reply:
done